backend/common/src/main/java/ai/verta/modeldb/common/CommonDBUtil.java

```diff
@@ -248,6 +256,22 @@ protected void createTablesLiquibaseMigration(
     }
   }
 
+  private static boolean changeLogTableExists(JdbcConnection jdbcCon, String changeLogTableName)
+      throws DatabaseException, SQLException {
+    boolean changeLogTableExists = false;
+    DatabaseMetaData metaData = jdbcCon.getMetaData();
+    ResultSet tables = metaData.getTables(null, null, "%", null);
+    while (tables.next()) {
+      String tableName = tables.getString(3);
+      if (changeLogTableName.equalsIgnoreCase(tableName)) {
```

Comment:
I think it might be a good idea to consider camelCase vs snake_case here, too.  I was looking back on the names that we had to deal with before, and [one was `DATABASECHANGELOG` and the other is `database_change_log`](https://github.com/VertaAI/verta-uacservice/blob/3324568498b17231ddeb0542e17ec5d184bb3745/src/main/java/ai/verta/uac/configuration/Migration.java#L26-L29). Not certain if that is applicable here, but wanted to mention it.

Reply:
hmm. the main purpose here was not to try to update a table that doesn't exist if we had an empty database. Do you have another idea about how to do that?

Reply:
if it turns out that the liquibase table (as configured) doesn't exist, then we shouldn't try to update it. So, I *think* this is a safe way to check.

Reply:
Cool 👍